projects/core/tsconfig.lib.json

```diff
@@ -11,7 +11,10 @@
     "experimentalDecorators": true,
     "importHelpers": true,
     "types": [],
-    "lib": ["esnext", "dom"]
+    "lib": ["esnext", "dom"],
+    "paths": {
```

Comment:
I think we should never reference incubator in the core, so that is not required.